lib/msee.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * `workshopper-adventure` doesn't offer any options to override its `print` command, and internal
+ * usage of `msee`, so we need to fork it into here so we can have wider Markdown screens.
+ *
+ * @license MIT
+ * @link https://github.com/workshopper/workshopper-adventure
+ */
+const msee = require('msee');
+
+const mseeOptions = {
+  paragraphStart: '',
+  paragraphEnd: '\n\n',
+  hrChar: '\u2500',
+  listItemPad: {
+    right: '   ',
+    first: '  » ',
+    regular: '    ',
+  },
+  defaultCodePad: '    ',
+  paragraphPad: {
+    left: ' ',
+    right: '   ',
+  },
+  maxWidth: 98, // 100 - 2 for the 2 character indent that bulletpoints add.
```

Comment:
Is it possible to take the full width of the screen?

Reply:
Looking through the `msee` code `-1` might do that?

https://github.com/firede/msee/blob/f2d3699076e058245b3c4acd9e40f2a69a58963c/lib/msee.js#L358-L360

Reply:
If so, it would be awesome, since folks usually like to control width by adjusting their terminal itself

Reply:
![Screen Shot 2021-08-13 at 10 26 18 AM](https://user-images.githubusercontent.com/33762/129397019-d22af886-d96c-42ee-b316-5fafe51187d8.png)

Yep -1 does it. Pushed that fix into this PR.